### DISCUSSION TITLE
Confirmation of patch for CVE issue.

### DISCUSSION THREAD

**Discussion by vlakken** [dexie/Dexie.js#2044] [General]
Hello I am currently using version 2.0.4 and would like to confirm that applying the following patch https://github.com/dexie/Dexie.js/commit/1d655a69b9f28c3af6fae10cf5c61df387dc689b is sufficient to fix the following issue https://security.snyk.io/vuln/SNYK-JS-DEXIE-2607042

**Comment by dfahlander** [comment]
I can only suggest to upgrade to dexie@3 or @4 which are maintained. I would assume that the commit you refer to would be the fix for that issue but if you want me to deep dive into that, you are welcome to [contact support](https://dexie.org/contact#private-support-issues)
